sections/semantics-forms.include

```diff
@@ -6888,49 +6754,49 @@ You cannot submit this form when the field is incorrect.</samp></pre>
+  1. If the <{input/step}> attribute is absent, then the <a>allowed value step</a> is the
       <a>default step</a> multiplied by the <a>step scale factor</a>.
-  2. Otherwise, if the attribute's value is an <a>ASCII case-insensitive</a> match for the string 
+  2. Otherwise, if the attribute's value is an <a>ASCII case-insensitive</a> match for the string
       "<code>any</code>", then there is no <a>allowed value step</a>.
-  3. Otherwise, let <var>step value</var> be the result of running the 
-      <a>rules for parsing floating-point number values</a>, when they are applied to the 
+  3. Otherwise, let <var>step value</var> be the result of running the
+      <a>rules for parsing floating-point number values</a>, when they are applied to the
       <{input/step}> attribute's value.
-  4. If the previous step returned an error, or <var>step value</var> is zero, or a number less than 
-      zero, then the <a>allowed value step</a> is the <a>default step</a> multiplied by the 
+  4. If the previous step returned an error, or <var>step value</var> is zero, or a number less than
+      zero, then the <a>allowed value step</a> is the <a>default step</a> multiplied by the
       <a>step scale factor</a>.
-  5. If the element's <{input/type}> attribute is in the 
+  5. If the element's <{input/type}> attribute is in the
       <a element-state for="input">Date and Time</a>,
       <a element-state for="input">Date</a>,
       <a element-state for="input">Month</a>,
       <a element-state for="input">Week</a>, or
       <a element-state for="input">Time</a> state, then round <var>step value</var> to the nearest
-      whole number using the "round to nearest + round half up" technique, unless the value is 
+      whole number using the "round to nearest + round half up" technique, unless the value is
       less-than one, in which case let <var>step value</var> be 1.
-  6. The <a>allowed value step</a> is <var>step value</var> multiplied by the 
+  6. The <a>allowed value step</a> is <var>step value</var> multiplied by the
       <a>step scale factor</a>.
 
   The <dfn>step base</dfn> is the value returned by the following algorithm:
 
-  1. If the element has a <{input/min}> content attribute, and the result of applying the 
+  1. If the element has a <{input/min}> content attribute, and the result of applying the
       <a>algorithm to convert a string to a number</a> to the value of the <{input/min}> content
       attribute is not an error, then return that result and abort these steps.
-  2. If the element does not have a <code>multiple</code> attribute specified or if the 
-      <code>multiple</code> attribute <a>does not apply</a>, then: if the element has a 
-      <code>value</code> content attribute, and the result of applying the 
-      <a>algorithm to convert a string to a number</a> to the value of the <{input/value}> content 
+  2. If the element does not have a <code>multiple</code> attribute specified or if the
```

Comment:
Good catch. These conditions look specific to multiple ranges. Let me edit this...